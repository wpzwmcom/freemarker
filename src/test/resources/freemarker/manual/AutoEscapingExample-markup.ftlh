<#ftl output_format="HTML"><#-- Needed for the Eclipse plugin -->
<#assign markupOutput1="<b>Test</b>"?no_esc>
<#assign markupOutput2="Foo & bar"?esc>

As expected:
${markupOutput1}
${markupOutput2}

Double escaping:
${markupOutput1?markup_string}
${markupOutput2?markup_string}