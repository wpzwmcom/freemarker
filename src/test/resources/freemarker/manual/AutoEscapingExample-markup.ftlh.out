
As expected:
<b>Test</b>
Foo &amp; bar

Double escaping:
&lt;b&gt;Test&lt;/b&gt;
Foo &amp;amp; bar